conf/broker.conf

```diff
@@ -582,7 +580,7 @@ delayedDeliveryMaxIndexesPerBucketSnapshotSegment=5000
 delayedDeliveryMaxNumBuckets=-1
 
 # Size of the lookahead window to use when detecting if all the messages in the topic
-# have a fixed delay.
+# have a fixed delay for InMemoryDelayedDeliveryTracker (the default DelayedDeliverTracker).
```

Comment:
```suggestion
# have a fixed delay for InMemoryDelayedDeliveryTracker (the default value is DelayedDeliverTracker).
```
Do you mean this?

Reply:
No, the default DelayedDeliverTracker is `InMemoryDelayedDeliveryTracker`